datasets/atomic/README.md

```diff
@@ -0,0 +1,185 @@
+---
+annotations_creators:
+- crowdsourced
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- cc-by-4.0
+multilinguality:
+- monolingual
+size_categories:
+- 100K<n<1M
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+task_ids:
+- other-stuctured-to-text
+---
+
+# Dataset Card for An Atlas of Machine Commonsense for If-Then Reasoning - Atomic Common Sense Dataset
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:**
+https://homes.cs.washington.edu/~msap/atomic/
+- **Repository:**
+https://homes.cs.washington.edu/~msap/atomic/
+- **Paper:**
+Maarten Sap, Ronan LeBras, Emily Allaway, Chandra Bhagavatula, Nicholas Lourie, Hannah Rashkin, Brendan Roof, Noah A. Smith & Yejin Choi (2019). ATOMIC: An Atlas of Machine Commonsense for If-Then Reasoning. AAAI
+
+### Dataset Summary
+
+This dataset provides the template sentences and
+relationships defined in the ATOMIC common sense dataset. There are
+three splits - train, test, and dev.
+
+From the authors.
+
+Disclaimer/Content warning: the events in atomic have been
+automatically extracted from blogs, stories and books written at
+various times. The events might depict violent or problematic actions,
+which we left in the corpus for the sake of learning the (probably
+negative but still important) commonsense implications associated with
+the events. We removed a small set of truly out-dated events, but
+might have missed some so please email us (msap@cs.washington.edu) if
+you have any concerns.
+
+
+For more information, see: https://homes.cs.washington.edu/~msap/atomic/
+
+### Languages
+en
+
+## Dataset Structure
+
+### Data Instances
+
+The atomic dataset has the following fields:
+
+
+``
+{
+	'event': ...,
+	'oEffect': ...,
+	'oReact': ...,
+	'oWant': ...,
+	'xAttr': ...,
+	'xEffect': ...,
+	'xIntent': ...
+	'xNeed': ...
+	'xReact': ...
+	'xWant': ...
+	'prefix': ...
+	'split': ...
+}
+``

```

Comment:
Can you provide an actual example from the dataset ?

You can get one by using `load_dataset` and then taking the first sample `dataset["train"][0]` for example

Reply:
Ok. I'll add this.

Reply:
Done